webpack.common.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+module.exports = {
+  entry: {
+    'index': './src/js/index.js',
+    'extras/media-clip-selector/index': './src/js/extras/media-clip-selector/index.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        loader: 'esbuild-loader',
+        options: {
+          target: 'es2015',
+        },
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.js'],
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: '[name].js',
+    library: {
+      type: 'module',
+    },
+  },
```

Comment:
Does this all the work the same way it used to? Basically everything in `/src` should be transpiled and output to `dist`, with 2 additional files that include All The Things (`/index.js` and `/extras.js`).

The goal is for people to be able to just `import 'media-chrome'` and get the kitchen sink, or be able to just import the individual elements they want to use. Same for extras. If I'm recalling Webpack correctly, this is going to output 2  bundles based on the `entry` key.

Reply:
err I meant to do "request changes" not "approve" because I think my output script question is still pretty important.

Reply:
per chat with @heff - my concern with that goal is it's very likely that people will accidentally end up with duplicate modules (some bundled, some unbundled) which could lead to both bloat and unexpected behaviors. Thoughts?

Reply:
Well if the kitchen sink bundle is an ES Module, it's just importing everything anyway, so I don't _think_ duplication would do anything there since imports are only imported once. Right?

Reply:
I'm currently creating modules that are also bundles, but I can def refactor our build to keep each module independent instead (different tradeoffs from a lib perspective).